Makefile

```diff
@@ -23,4 +23,4 @@ install-bin: all
 install-systemd: all
 	install -D -m 644 -t $(DESTDIR)/usr/lib/systemd/system systemd/*.{service,target}
 	install -D -t $(DESTDIR)/usr/lib/systemd/system-generators systemd/coreos-installer-generator
-	install -D -t $(DESTDIR)/usr/libexec systemd/coreos-installer-service
+	install -D -t $(DESTDIR)/usr/libexec systemd/coreos-installer-devices systemd/coreos-installer-service
```

Comment:
I have a feeling both of these shouldn't be under `systemd/`, as it can only grow messier from here on.